src/app.rs

```diff
@@ -299,13 +317,32 @@ impl App {
                         ctx.draw(&line);
                     }
                 }
-                for line in get_frame_lines(
-                    tf_listener.clone(),
-                    self.static_frame.clone(),
-                    self.axis_length,
-                ) {
+
+                for line in get_frame_lines(&base_link_pose, self.axis_length) {
                     ctx.draw(&line);
                 }
+                if !poses_are_close(&self.initial_pose, &self.pose_estimate) {
+                    let pose_estimate_ros = transformation::minimal_to_ros(
+                        self.pose_estimate.0,
+                        self.pose_estimate.1,
+                        self.pose_estimate.2,
+                    );
+                    let ghost_footprint =
```

Comment:
i never know but do we need the assignment here?

Reply:
I guess not